computation_dag/topology_builder.py

```diff
@@ -15,8 +16,14 @@ def _validate_path(self):
 
     def _try_load(self, f, **kwargs):
         try:
-            df = f(path=self.path, **kwargs)
-            df.first()
+            if isinstance(self.path, str):
```

Comment:
Better: `isinstance(self.path, basestring)`

https://stackoverflow.com/questions/1303243/how-to-find-out-if-a-python-object-is-a-string